### ISSUE TITLE
Chromium crash with DEBUG on (-v flag)

### ISSUE THREAD

**Issue by klichukb** [adieuadieu/serverless-chrome#168]
Was having a crash in 69.0.3497.81 while having DEBUG on.
Tried both real AWS lambda and locally built Docker image.
Commenting out "-v=99" flag (or any verbosity, even 0) helps.
 
```
DevTools listening on ws://127.0.0.1:9222/devtools/browser/ce7bda0f-abe2-4297-bd33-55fe2f4e3e24
[1002/214435.266159:VERBOSE1:proxy_resolution_service.cc(1016)] PAC support disabled because there is no system implementation
[1002/214435.277906:VERBOSE2:thread_state.cc(621)] [state:0x6aecca0] ScheduleGCIfNeeded
Received signal [1002/214435.331351:VERBOSE2:thread_state.cc(621)] [state:0x6aecca0] ScheduleGCIfNeeded
[1002/214435.331639:VERBOSE2:thread_state.cc(621)] [state:0x6aecca0] ScheduleGCIfNeeded
11 SEGV_MAPERR 000000000000
#0 0x000003b7633c <unknown>
#1 0x000003b75ea1 <unknown>
#2 0x7fd38f4085a0 <unknown>
#3 0x7fd38de96f7f __strlen_sse42
#4 0x000004810bf8 <unknown>
#5 0x000004810728 <unknown>
#6 0x000004897c58 <unknown>
#7 0x000003b07079 <unknown>
#8 0x000003b06262 <unknown>
#9 0x000003b06727 <unknown>
#10 0x000003b87529 <unknown>
#11 0x000003b1f245 <unknown>
#12 0x000003613a64 <unknown>
#13 0x000003613823 <unknown>
#14 0x000003616052 <unknown>
#15 0x000003b9857f [1002/214435.340311:VERBOSE2:thread_state.cc(621)] [state:0x6aecca0] ScheduleGCIfNeeded
<unknown>
#16 0x000003ab132e <unknown>
#17 0x000004bfefcb <unknown>
#18 0x000003aa9d31 <unknown>
#19 0x000003b9745d <unknown>
#20 0x000003b974e8 <unknown>
#21 0x000003aea99a <unknown>
#22 0x7fd38dd7bc05 __libc_start_main
#23 0x00000242402a _start
  r8: 0000000000000000  r9: 00001cfd85aeac68 r10: 00007fd38de8f790 r11: 0000000000000000
 r12: 00007ffdb050e7f0 r13: 00001cfd85a3da00 r14: 00001cfd85a3dd88 r15: 00001cfd859f2340
  di: 0000000000000000  si: 0000000000000000  bp: 00007ffdb050e950  bx: 0000000000000000
  dx: 0000000006affc10  ax: 72657265646e6552  cx: 0000000000000000  sp: 00007ffdb050e6c8
  ip: 00007fd38de96f7f efl: 0000000000010246 cgf: 002b000000000033 erf: 0000000000000004
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000000
[end of stack trace]
Calling _exit(1). Core file will not be generated.
```

As soon as verbosity flag is removed or DEBUG turned off, this works well.

**Comment by timoa** [comment]
Thanks! That fixed the issue #170 for me :)

**Comment by python1981** [comment]
I would up-vote this 100 times if I could.
